documentation/modules/non-admin-permissions-for-ui.adoc

```diff
@@ -0,0 +1,56 @@
+:_content-type: PROCEDURE
+
+[id="non-admin-permissions_{context}"]
+== Granting permissions to non-administrators to work with migration plan components
+
+As an administrator, you can work with all components of migration plans (for example, providers, network mappings, and migration plans).
+
+By default, non-administrators have limited ability to work with migration plans and their components. You can modify their roles to allow them full access to all components or you can give them limited permissions.
+
+For example, you can grant non-administrators the following permissions:
+
+* Create and modify storage maps, network maps, and migration plans for the namespaces they have access to
+* View, but not create, providers and attach them to components as needed
+
+[cols="3", options="header"]
+.Permissions required for non-adminstrators to work with migration plan components but not create providers
+|===
+|Actions |API groups |Resources
+
+|All
+|`forklist.conveyer.io`
+|`plans`
+
+|All
+|`forklist.conveyer.io`
+|`migrations`
+
+|All
+|`forklist.conveyer.io`
+|`hooks`
+
+|`get`, `list`, `watch`
+|`forklist.conveyer.io`
+|`providers`
+
+|All
+|`forklist.conveyer.io`
+|`networkmaps`
+
+|All
+|`forklist.conveyer.io`
+|`storagemaps`
+
+|`get`, `list`, `watch`
+|`forklist.conveyer.io`
+|`forkliftcontrollers`
+|===
+
```

Comment:
ok, we can go with a single concrete example but then please add another note that users need to have 'create' permissions on network maps and storage maps in order to create a migration plan even when using template mappings

Reply:
Note added.